microhab/SPEXpdd.tex

```diff
@@ -0,0 +1,253 @@
+    Rochester, N.Y. \\
+    Email:
+    \IEEEauthorrefmark{1}abb6499@rit.edu
+  }
+}
+% page header for pages other than cover page
+\markboth{\textmu{}hab}%
+{Bodzas \MakeLowercase{\textit{et al.}}: RIT Space Exploration}
+
+% Initial setup is over, start building the document itself
+\begin{document}
+\maketitle%
+% correct bad hyphenation here, separated by spaces
+\hyphenation{explor-ation}
+
+\begin{abstract}
+
+      % The abstract is a brief summary of the design document. Typically it includes the purpose of the design document, key goals or objectives, and justifications.
+      % Be sure not to confuse the abstract with the introduction.
+      % It is easiest to write the abstract after the rest of the paper has been written.
+      % That way you can choose key information from the sections that you've already completed and string them together in the abstract.
+      % Consider the abstract to be your elevator pitch to anyone reading this design document.
+      % What are they reading?
+      % What is the goal?
+      % Why is it worth my time?
+      % The abstract is what will show up in Google results and other search engines, and what people will read when they are deciding what is worth their time and brain power.
+    A project definition for the \textmu{}Hab project.  \textmu{}Hab is the term the
+    HAB team has been using to describe a smaller balloon and payload than is
+    typically used by the group (1200 grams). However, this project expands upon
+    the term with the main goal of minimizing costs that go into building and
+    launching a HAB.\@
+\end{abstract}
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{}
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{HAB}{High Altitude Balloon}
+% Below are examples of using nomenclature for math symbols and constants or units
+% \nomenclature{$\dot{m}$}{Mass flow rate
+%   \nomunit{\,\si{\kilo\gram\per\second}}}
+% \nomenclature{$c$}{Speed of light
+%  \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+
+% HELPFUL HINT: If you get the warning ``Command terminated with space.'' when using a \command try placing ``%'' or ``{}'' immediately following the command.
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{S}{PEX} has launched three High Altitude Balloons (HABs) to date.  The
+motive for a launch is to educate students on engineering processes, test experiments in
+a near space-like environment, and to have fun in the process. The HAB team within SPEX aims to
```

Comment:
I understand that having fun is part of the project, but is it an objective that you're trying to meet? I'll have to keep reading but it sounds like you're establishing the important parts of a HAB launch to preface how microHab will also meet those objectives